conda_libmamba_solver/solver.py

```diff
@@ -314,21 +315,42 @@ def _solving_loop(
 
     def _print_info(self):
         if not context.json and not context.quiet:
-            print(
-                dedent(
-                    f"""
-                    ----       USING EXPERIMENTAL LIBMAMBA INTEGRATIONS       ----
-                        This is a highly experimental product. If something is
-                        not working as expected, please submit an issue at
-                        https://github.com/conda/conda and attach the log file
-                        found in the following path. Thank you!
-
-                        {context._logfile_path}
+            msg = dedent(
+                f"""
+                ----       USING EXPERIMENTAL LIBMAMBA INTEGRATIONS       ----
+                    This is a highly experimental product. If something is
+                    not working as expected, please submit an issue at
+                    https://github.com/conda/conda and attach the log file
+                    found in the following path. Thank you!
+
+                    {context._logfile_path}
+
+                ---------------------------------------------------------------
+                """
+            ).rstrip()
+            if context.experimental_solver_confirmation:
+                risk = dedent(
+                    """
+                    There are risks using this experimental solver since it
+                    does not always produce identical results as that of the
+                    classical solver. As with any experimental/beta product it
+                    is not recommended to use this in production/critical
+                    environments.
```

Comment:
```suggestion
                    There are risks using this experimental solver since it
                    does not always produce identical results as that of the
                    classical solver. As with any experiment it is *not*
                    recommended to use it in production or critical
                    environments.
```

We've tried to not use the "beta" terminology since it's perceived differently by users, really hard to understand what it means, while experiment is intentional about it being an unfinished product.